playing/getting_started/index.rst

```diff
@@ -0,0 +1,26 @@
+Getting Started
+===============
+
+In OpenRCT2, there are two main game modes - Scenario mode and Sandbox mode.
```

Comment:
This suggests that there is a technical distinction between the two, which is not the case. There is only scenario mode, although you can adjust the settings in a way that makes it more like a sandbox. Please rephrase this section and the one about "playing a sandbox" to emphasise that better.

A sandbox mode is a nice-to-have that is on our list (though not super high), which is probably why it ended up in the TOC, but the documentation should reflect the _current_ state of the game.

Reply:
Sounds good, would you want to just delete the section on Sandboxes then? It doesn't seem to add that much on its own.

Or should I mention something about enabling cheats and getting rid of money/objectives?

Reply:
Yeah, mention the following:
- all games are scenarios with an objective
- a sandbox mode is planned, in the meantime one can use cheats\* to emulate it

and remove the separate section about sandboxes.

\* we should turn that into a link once there is a page about using cheats